index.html

```diff
@@ -134,11 +134,11 @@
                     Courses
                 </a>
                 <div class="dropdown-menu text-dark" aria-labelledby="navbarDropdownMenuLink">
-                    <a class="dropdown-item text-dark" href="overview.html">Overview</a>
-                    <a class="dropdown-item text-dark" href="overview.html#science">Science</a>
-                    <a class="dropdown-item text-dark" href="overview.html#math">Math</a>
-                    <a class="dropdown-item text-dark" href="overview.html#compsci">Computer Science</a>
-                    <a class="dropdown-item text-dark" href="overview.html#inter">Interdisciplinary</a>
+                    <a class="dropdown-item text-dark" href="overview.html#overview#undefined">Overview</a>
```

Comment:
I forgot why the #undefined happens but we should fix it instead of changing the hrefs since it doesn't look good in the url to users anyway

Reply:
It happens because the js needs two fragments, a subject and class. If you only provide the subject, it will set the class to undefined.

The best option would probably be to change the hrefs from #undefined to e.g. #general, or some other tag.

Reply:
Looks like I *tried* to filter this out with `location.hash = "#" + _class == null ? subject : subject + "#" + _class;` but class is being passed as "undefined" instead of null so we should check against that instead. Haven't tested but should fix it I think